spec/std/uuid_spec.cr

```diff
@@ -0,0 +1,150 @@
+require "spec"
+require "uuid"
+
+describe "UUID" do
+  describe "default initialize" do
+  end
```

Comment:
This `describe` block is empty. Was this intentional?

Reply:
Thanks for pointing this out.
I have updated it.